t/api/05-machines.t

```diff
@@ -163,6 +163,69 @@ is_deeply(
 $res = $t->delete_ok("/api/v1/machines/$machine_id")->status_is(200);
 $res = $t->delete_ok("/api/v1/machines/$machine_id")->status_is(404);    #not found
 
+subtest 'add and update machine with white spaces' => sub {
+    $res = $t->post_ok(
```

Comment:
Remember you don't need the `$get` or `$res` variables here. `$t->tx` always has the result of the transaction.

Reply:
> I think you need to rebase after we have updated the code style in another PR

Before I committed code, I have already rebased the code and updated the perl tidy version.